style.scss

```diff
@@ -0,0 +1,631 @@
+                grid-row: 1 / 3;
+            }
+
+            &--ui-ux {
+                grid-column: 2 / 3;
+                grid-row: 1 / 2;
+            }
+
+            &--apps {
+                grid-column: 3 / 4;
+                grid-row: 1 / 2;
+            }
+
+            &--illustrations {
+                grid-column: 2 / -1;
+                grid-row: 2 / 3;
+            }
+
+            &--photography {
+                grid-column: 1 / 2;
+                grid-row: 3 / 4;
+            }
+
+            &--motion-graphics {
+                grid-column: 2 / 4;
+                grid-row: 3 / 4;
+            }
+        }
+    }
+    
+    .portfolio__about-me{
+        position: relative;
+        padding-block: var(--size-1200);
+        
+        &-image {
+            position: relative;
+            left: -12%;
+        }
+        
+        &-intro {
+            justify-content: center;
+            gap: var(--size-500);
+            max-width: 21.1875rem;
+        }
+        
+    }
+    
+
+    .portfolio__carousel{
+        padding-block: 0;
+        padding-bottom: var(--size-1200);
+        
+        &-slider {
+            margin-top: 2rem;
+            margin-bottom: 2rem;
+            
+            &>img{
+                border-radius: 0.625rem;
+                width: 33.75rem;
+            }
+        }
+    }
+    
+    .portfolio__booking {
+        padding: var(--size-750) 4.6875rem var(--size-800) 4.6875rem;
+    }
+            
+    .footer{
+        padding: var(--size-750) 0;
+    }
+}
+
+@media only screen and (min-width: 1110px) {
+    
+    :root {
+        --fs-body: var(--fs-450);
+        --fs-small-heading: var(--fs-450);
+        --fs-medium-heading: var(--fs-800);
+        --fs-large-heading: var(--fs-900);
+    }
+    
+    .portfolio__intro {
+        padding-block: var(--size-600) var(--size-900);
+        .description {
+            padding: 0 11.875rem;
+        }
+    }
+    
+    .portfolio__skills {            
+        &-grid {
+            grid-template-columns: minmax(1rem, 2fr) minmax(1rem, 1fr) minmax(1rem, 1fr) minmax(1rem, 2fr);
+            grid-template-rows: repeat(2, 1fr);
+        }
+        
+        &-grid-item {
+            &--illustrations {
+                grid-column: 2 / 4;
+                grid-row: 2 / 3;
+            }
+
+            &--photography {
+                grid-column: 4 / -1;
+                grid-row: 1 / 2;
+            }
+            
+            &--motion-graphics {
+                grid-column: 4 / -1;
+                grid-row: 2 / 3;
+            }
+        }
+    }
+    
+    .portfolio__about-me{
+        gap:7.8125rem;
+        
+        &-image {
+
+            max-width: 27.8125rem;
+            left: 0;
+        }
+        
+        &-intro {
+            max-width: 33.75rem;
+            gap: 2.125rem;
+        }
+    }
+    
+    .portfolio__booking {
+        flex-direction: row;
+        gap: 0;
+        
+        &-info{
```

Comment:
There is nor element, nor modifier. It could be:
portfolio__booking and portfolio__booking-info (as a separate element)
Or portfolio__booking--info (as modifier)